src/openxr/OpenXRApi.cpp

```diff
@@ -2968,7 +3079,7 @@ void OpenXRApi::render_openxr(int eye, uint32_t texid, bool has_external_texture
 		return;
 	}
 
-	if (!has_external_texture_support) {
+	if (!has_external_texture_support && !has_external_depth_support) {
```

Comment:
This bit of code has always been here to ensure we have some backwards compatibility with some of the earlier versions of Godot. I'm wondering if we should remove it all together. It would result in the the plugin no longer working on Godot 3.3 and earlier, or whatever version of Godot i added depth texture support to. It's been in there awhile even if we never implemented it in the plugin before.

Reply:
According to our docs, this plugin is only supported starting with Godot 3.4. So if this code only affects Godot 3.3 and earlier, then yes we can remove it.

Reply:
By the looks of things, Godot 3.2 and earlier. I think I added the logic in 3.3 :)

Ok I cleaned it up, no use holding on to old code.